evento/src/test-helpers/mock-factory.js

```diff
@@ -6,6 +6,13 @@ const generateId = () =>
 const generateString = (n) =>
 	(Math.random().toString(36) + '00000000000000000').slice(2, n + 2);
 
+// Return future time stamp
+const generateTimeStamp = (daysInFuture=2) => {
```

Comment:
dayOffset could be better, to allow past days as well. Add spaces around the param as well